controllers/article-controllers.js

```diff
@@ -0,0 +1,22 @@
+const { fetchArticle, updateArticle } = require("../models/article-models");
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticle(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
+    })
+    .catch(next);
+};
+
+exports.patchArticle = (req, res, next) => {
+  const articleId = req.params;
+  //console.log(req.body, "REQ BODY");
```

Comment:
Make sure you remove any unnecessary console logs after you merge this, console.log's are for debugging and trouble shooting, if this code is ready to be merged, you should be done with that :)